runtime/oti/vmconstantpool.xml

```diff
@@ -128,6 +128,7 @@
 	<classref name="java/lang/Module" jcl="se9,se10,se11" flags="opt_module"/>
 	<classref name="java/lang/invoke/VarHandle" jcl="se9_before_b165,se9,se10,se11"/>
 	<classref name="java/lang/invoke/VarHandleInvokeHandle" jcl="se9_before_b165,se9,se10,se11"/>
+	<classref name="java/io/IOException" />
```

Comment:
Its better to use the JNI methods to look up the class rather than add this to the vm cp